LibraBFT/Impl/OBM/Prelude.agda

```diff
@@ -1,31 +0,0 @@
-{- Byzantine Fault Tolerant Consensus Verification in Agda, version 0.9.
```

Comment:
@cwjnkins, you resolved the conversation, but didn't record why.  I think it's because you replaced these with Agda standard library functions, but I tend to agree that we should keep this faithful to the implementation (e.g., suppose we got the definitions wrong in the implementation, but using the correct version from the Agda library masked the issue).